### ISSUE TITLE
admin/statistic

### ISSUE THREAD

**Issue by smadamit** [inbaz/ers#383]
v0.3.33

wrong -> Participants count:
- counts persons, which buy only a personalized galaticket, but without day ticket / week ticket

rest of the statistic works fine

**Comment by inbaz** [comment]
On the statistics page the participant count is counting even orders which only include a gala show tickets. A more detailed participant statistic would be nice to see how much week tickets and how much day tickets on each day were sold. Maybe the participant count with week tickets are enough because day count is included in the variant statistics.

**Comment by inbaz** [comment]
Please add a table on how the statistic should look like and which numbers should be included. I will add that on the separate participant statistics page.

**Comment by smadamit** [comment]
let's discuss in Skype. See also #392

**Comment by inbaz** [comment]
What was the quintessence of the discussion on this issue? There are many alarm bells ringing in my head when statistic and bug is in the headline of an issue. So if there is not really a bug, close this issue and we'll work on all other tasks in the other issues.

**Comment by smadamit** [comment]
it's a very week bug, maybe it depends on the point of view weather it is a bug or not. I put it to AfterEJC. Not important for now.

This is my issue: counts persons, which buy only a personalized galaticket, but without day ticket / week ticket

**Comment by inbaz** [comment]
So what you want is to exclude "participants" who only own a gala ticket because they do not get a badge and are therefore not really onsite, right?

**Comment by smadamit** [comment]
Yes, my understanding of a EJC participant is a person, who orders a week ticket or one/serveral day ticket/s. But not important for now. Changes the participant-count maybe by 20-50...

**Comment by inbaz** [comment]
Is this solved with the statistic from @Bone008?

**Comment by Bone008** [comment]
For now, I kept the definition that participants are people with whatever kind of ticket - including show tickets. I think what is more appropiate should be discussed after EJC.

**Comment by inbaz** [comment]
@jolieno This is not urgent at the moment but I'll give the ticket to you. Maybe you just read through the discussion of last year and think of how your statistics should behave. My suggestion would be that there will ever be a small error within the statistics but for a quick overview it doesn't matter if the people who only buy a gala show ticket are counted als real participants. However the count of week and day tickets is also next to this full number, therefore the real number can be easily calculated. If that works for you we can close this ticket.

**Comment by inbaz** [comment]
Maybe a small field with an explanation of what the numbers include would be nice. First of all this field needs to be prepared to be able to add new elements easily.
